### PR TITLE
add 15551 and zukeswap

### PR DESCRIPTION
**Pull request by ZukeSwap**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### New Blockchain Submissions:
We'll need the following to integrate your **EVM-compatible** blockchain: 

- [x] At least one AMM on your chain with a daily volume over $100k. Do you include it in your PR?   
- [x] Your chain logo
- [x] Your native token logo
- [x] Blockexplorer that could open account and transactions pages 
- [x] High-performance RPC endpoint. Raise the issue in this repo or reach out our team at [Discord](https://discord.com/invite/dPW8fzwzz9) with access details.  

### New AMM Submissions:
We'll need the following to integrate your AMM: 

* [x] Do you have a daily trading volume over $20k? 
* [x] Does "type" at AMM config match your code base? 
* [x] Your AMM logo. Please add a link to your logo or attach it to a new issue in this repo.

### CODE REVIEW

**Comment by ZukeSwap** [comment]
Chain & Native Token Logo: https://raw.githubusercontent.com/ZukeSwap/zukeswap-default-token-list/main/assets/WLOOP/logo.png

AMM logo: https://raw.githubusercontent.com/ZukeSwap/zukeswap-default-token-list/main/assets/0xE3bAAb444d70CE9FCe8d07Ef24Af1Ae5C5E09fD1/logo.png

File: dapps/evm/ZukeSwap/config.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[
+    {
+      "chain_id": 15551,
+      "name": "zukeswap",
+      "type": "zukeswap",
```
Comment (evahteev) [inline dapps/evm/ZukeSwap/config.json:5]: type: uniswap_v2

File: chains/evm/15551/config.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      "type": "evm",
+      "enabled": true,
+      "native_token_address": "0x3936D20a39eD4b0d44EaBfC91757B182f14A38d5",
+      "native_token": {
+        "address": "0x3936D20a39eD4b0d44EaBfC91757B182f14A38d5",
+        "name": "Wrapped LOOP",
+        "symbol": "WLOOP",
+        "decimals": 18
+      },
+      "stablecoin_addresses": []
```
[the remark this answers is not in the document]
Comment (ZukeSwap) [reply chains/evm/15551/config.json:14]: Stablecoins is not available yet on this chain

[the remark this answers is not in the document]
Comment (evahteev) [reply chains/evm/15551/config.json:14]: I'm sorry we can't integrate without them

[the remark this answers is not in the document]
Comment (ZukeSwap) [reply chains/evm/15551/config.json:14]: okay I understand

**Comment by evahteev** [comment]
Due to Lack of stablecoins on chain.

**Pull request closed without merging**